org/README.md

```diff
@@ -0,0 +1,9 @@
+# Tektoncd Github Organization
+
+This directory contains the configuration of the Tektoncd Github organization.
+To add or remove members and admins, make changes to this file.
+
+This system uses [peribolos](https://github.com/kubernetes/test-infra/tree/master/prow/cmd/peribolos)
+to manage the org configuration.
+
+After config updates are made, an administrator must manually sync the configurations for them to take effect.
```

Comment:
I assume the manual sync part is just temporary?

Reply:
Yeah, I just need to get both halves setup, which I can't do until this is in.